Cargo.lock

```diff
@@ -2898,7 +2897,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0c2a198fb6b0eada2a8df47933734e6d35d350665a33a3593d7164fa52c75c19"
 dependencies = [
  "cfg-if",
- "windows-targets 0.52.4",
+ "windows-targets 0.48.5",
```

Comment:
Ugh, this was _just_ bumped by https://github.com/ruffle-rs/ruffle/pull/15777.
I don't care, shouldn't matter...
Super weird, hopefully DependaBot (#14146) won't do this nonsense.